sites/all/themes/wille/template.php

```diff
@@ -112,18 +112,22 @@ function wille_ting_collection_view_alter(&$build) {
  * Wraps the cover in a link to the material.
  */
 function wille_ting_object_cover($variables) {
+  $ding_entity_id = $variables['elements']['#object']->ding_entity_id;
+  $type = $variables['meta_for_labels']['type'];
```

Comment:
You may need to check that `$variables['meta_for_labels']` (and the sub keys `type` and `on_quota`) is set.